src/vm/gdbjit.cpp

```diff
@@ -1333,15 +1565,23 @@ int NotifyGdb::GetFrameLocation(int nativeOffset, char* bufVarLoc)
     return len + 2;  // We add '2' because first 2 bytes contain length of expression and DW_OP_fbreg operation.
 }
 /* Build tge DWARF .debug_info section */
-bool NotifyGdb::BuildDebugInfo(MemBuf& buf, NewArrayHolder<ArgsDebugInfo> &argsDebug, unsigned int argsDebugSize,
+bool NotifyGdb::BuildDebugInfo(MemBuf& buf, PMT_TypeInfoMap pTypeMap, NewArrayHolder<ArgsDebugInfo> &argsDebug, unsigned int argsDebugSize,
                                NewArrayHolder<LocalsDebugInfo> &localsDebug, unsigned int localsDebugSize)
```

Comment:
Can you please pass in the parameter las LocalsDebugInfo* instead? It seems we don't need to pass reference to the whole holder and it puts an implementation detail to the method signature.